### PR TITLE
Correcting dotnet install script paths: dot.net

### PR DESCRIPTION
**Pull request by ghost**
Correcting dotnet install script paths.

* https://dot.net/v1/dotnet-install.sh (bash, UNIX)
* https://dot.net/v1/dotnet-install.ps1 (powershell, Windows)

@dotnet/dotnet-cli

### CODE REVIEW

File: docs/core/linux-prerequisites.md [hunk 1]
```diff
@@ -101,7 +101,10 @@ On Linux, there are two installer package choices:
 
 ### Scripting Installs with the .NET Core installer script
 
-The `dotnet-install` scripts are used to perform a non-admin install of the CLI toolchain and the shared runtime. You can download the scripts from the [CLI GitHub repo](https://github.com/dotnet/cli/tree/rel/1.0.0/scripts/obtain).
+The `dotnet-install` scripts are used to perform a non-admin install of the CLI toolchain and the shared runtime. You can download the scripts from:
+
+* https://dot.net/v1/dotnet-install.sh (bash, UNIX)
+* https://dot.net/v1/dotnet-install.ps1 (powershell, Windows)
```
Comment (mairaw) [inline docs/core/linux-prerequisites.md]: given that this is the Linux topic, I'd only list the bash version

**Review by mairaw — CHANGES_REQUESTED** [review]
Thanks @johnbeisner. LGTM. Left just one comment to be addressed.

**Comment by johnbeisner** [comment]
@mairaw
I think this one is ready to merge.

**Review by mairaw — APPROVED** [review]
Yes, it's ready. Thanks @johnbeisner!

**Pull request merged**